client/my-sites/checkout/checkout-thank-you/jetpack-thank-you-card.jsx

```diff
@@ -562,12 +561,10 @@ class JetpackThankYouCard extends Component {
 			return (
 				<div className="checkout-thank-you__jetpack-action-buttons">
 					<a
-						className={ classNames( 'button', 'thank-you-card__button', {
-							'is-placeholder': ! buttonUrl,
-						} ) }
-						href={ buttonUrl }
+						className={ classNames( 'button', 'thank-you-card__button' ) }
+						href={ '/start/rewind-credentials?blogid=' + this.props.siteId }
```

Comment:
This can be written as 
```js
href={ `/start/rewind-credentials?blogid=${ siteId }` }
```
It needs to destructure `siteId` in [line 542](https://github.com/Automattic/wp-calypso/pull/21767/files#diff-a256b914f277ea5af200ec47e3ffe7fdL542)